contributingGuides/TS_STYLE.md

```diff
@@ -124,7 +126,7 @@ type Foo = {
 
 <a name="d-ts-extension"></a><a name="1.2"></a>
 
-- [1.2](#d-ts-extension) **`d.ts` Extension**: Do not use `d.ts` file extension even when a file contains only type declarations. Only exceptions are `src/types/global.d.ts` and `src/types/modules/*.d.ts` files in which third party packages can be modified using module augmentation. Refer to the [Communication Items](#communication-items) section to learn more about module augmentation.
+- [1.2](#d-ts-extension) **`d.ts` Extension**: Do not use `d.ts` file extension even when a file contains only type declarations. Only exceptions are `src/types/global.d.ts` and `src/types/modules/*.d.ts` files in which third party packages and internal JavaScript modules can be modified using module augmentation. Refer to the [Communication Items](#communication-items) section to learn more about module augmentation.
```

Comment:
This suggests that `d.ts` can be used for internal  modules, but doesn't provide any guidance on when that would be permissible. When would we do this? Can we provide an example?

Reply:
do you meant that when the internal package written in JavaScript needs to be module-augmented, we use `*d.ts` in the `modules` directory?

Reply:
Maybe let's be more specific or add an example? @fabioh8010

Reply:
I changed `internal JavaScript module` to `JavaScript's built-in module`, I think makes more clear what we are referring too.

@roryabraham [Here](https://github.com/Expensify/App/blob/main/src/types/modules/electron.d.ts#L12) is an example of what we are talking about!

Reply:
@fabioh8010 I still don't fully understand what you mean by `JavaScript's built-in module`. Do you mean functions and constants that are available during run time (those provided by the run time environment)?

Reply:
@hayata-suenaga  I mean global objects like `window` (and its `Window` interface), built-in TS interfaces like `File`, `Event` or any other DOM interfaces available. We have some examples in [this file](https://github.com/Expensify/App/blob/main/src/types/modules/pusher.d.ts)!